src/eko/basis_rotation.py

```diff
@@ -68,15 +89,68 @@
 )
 """|pid| representation of :data:`evol_basis`."""
 
-non_singlet_pids_map = {"ns-": 10201, "ns+": 10101, "nsV": 10200}
+unified_evol_basis_pids = tuple(
+    [21, 22, 100, 101, 200, 201]
+    + [103 + 2, 203 + 2]  # d = +2
```

Comment:
please make your choice of `+2` (which is just fine) more visible - e.g. in a docstring somewhere (likewise for Sdelta and Vdelta)